apps-rendering/src/components/LiveBlocks/index.tsx

```diff
@@ -29,30 +31,85 @@ const contributorToBlockContributor = (
 
 // ----- Component ----- //
 interface LiveBlocksProps {
-	blocks: LiveBlock[];
+	blocks: LiveBlockType[];
 	format: ArticleFormat;
 	pageNumber: number;
+	pinnedPost: Option<LiveBlockType>;
 }
 
-const LiveBlocks: FC<LiveBlocksProps> = ({ blocks, format, pageNumber }) => {
-	const pinnedPost = fromNullable(blocks.find((b) => b.isPinned));
+interface LiveBlockProps {
+	block: LiveBlockType;
+	format: ArticleFormat;
+	isPinnedPost: boolean;
+	isOriginalPinnedPost: boolean;
+}
+
+const LiveBlock: FC<LiveBlockProps> = ({
+	block,
+	format,
+	isPinnedPost,
+	isOriginalPinnedPost,
+}) => {
+	return (
+		<LiveBlockContainer
+			id={block.id}
+			format={format}
+			blockTitle={block.title}
+			blockFirstPublished={Number(block.firstPublished)}
+			blockId={block.id}
+			isPinnedPost={isPinnedPost}
+			isOriginalPinnedPost={isOriginalPinnedPost}
+			supportsDarkMode={true}
+			contributors={block.contributors.map(contributorToBlockContributor)}
+		>
+			{renderAll(format, partition(block.body).oks)}
+
+			<footer
+				css={css`
+					display: flex;
+					justify-content: end;
+				`}
+			>
+				{block.lastModified > block.firstPublished && (
+					<LastUpdated
+						lastUpdated={block.lastModified}
+						lastUpdatedDisplay={formatUTCTimeDateTz(
+							block.lastModified,
+						)}
+					/>
+				)}
+			</footer>
+		</LiveBlockContainer>
+	);
+};
 
+const LiveBlocks: FC<LiveBlocksProps> = ({
+	blocks,
+	format,
+	pageNumber,
+	pinnedPost,
+}) => {
 	const showPinnedPost =
 		pageNumber === 1 && pinnedPost.kind === OptionKind.Some;
 
 	return (
 		<>
 			{/* Accordion? */}
-			{/* below is placeholder for pinned post component */}
-			{showPinnedPost && <></>}
+			{showPinnedPost && (
+				<PinnedPost pinnedPost={pinnedPost.value} format={format}>
+					<LiveBlock
+						block={pinnedPost.value}
+						format={format}
+						isPinnedPost={true}
+						isOriginalPinnedPost={false}
+					></LiveBlock>
```

Comment:
You could use a self-closing tag for this:

```suggestion
					/>
```